src/service/app/statisticsChart.ts

```diff
@@ -69,6 +74,17 @@ const handleStatistics = (wrapper: Response.Wrapper<State.StatisticsChart>) => {
   return datas
 }
 
+const handleStatisticsUnlceRate = (wrapper: Response.Wrapper<State.StatisticsChart>) => {
+  const { uncleRate: uncleRates } = wrapper.attributes
```

Comment:
Why not just use `uncleRates` in `wrapper.attributes`?

Reply:
@shaojunda Maybe this interface can be changed to `uncle_rates`

Reply:
uncle rate is a piece of data in the statistic info. so I don't think it needs to be modified.